samples/add-raster-from-file/README.md

```diff
@@ -0,0 +1,34 @@
+# Add raster from file
+
+Create and use a raster layer made from a local raster file.
+
+![Image of add raster from file](add-raster-from-file.png)
+
+## Use case
+
+Rasters can be digital aerial photographs, imagery from satellites, digital pictures, or even scanned maps. An end-user will frequently need to import raster files acquired through various data-collection methods into their map to view and analyze the data.
+
+## How to use the sample
+
+When the sample starts, a raster will be loaded from a file and displayed in the map view.
+
+## How it works
+
+1. Create a `Raster` from a raster file.
+2. Create a `RasterLayer` from the raster.
+3. Add it as an operational layer with `map.getOperationalLayers().add(rasterLayer)`.
+
+## Relevant API
+
+* Raster
+* RasterLayer
+
+## Additional information
+
+See the topic [What is raster data?](http://desktop.arcgis.com/en/arcmap/10.3/manage-data/raster-and-images/what-is-raster-data.htm) in the *ArcMap* documentation for more information about raster images.
+
+This sample uses the GeoViewCompose Toolkit module to be able to implement a Composable MapView.
+
+## Tags
+
+data, image, import, layer, raster, visualization
```

Comment:
Oh, one other thing, [the epic](https://devtopia.esri.com/runtime/kotlin/issues/4961) says to:
> Add the tags: `geoviewcompose`, `toolkit` to readme and metadata.json file

Reply:
Done